packages/kitsu-core/src/deepEqual/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Compare two objects equality 
+ *
+ * @name isDeepEqual
+ * @param {object} firstObject First Object
+ * @param {object} secondObject Second Object to compare with the first one
+ * @returns {boolean} A boolean that indicates if objects are equal
+ *
+ * @example <caption>Deep equality check</caption>
+ * isDeepEqual({
+ *   "firstName": "John",
```

Comment:
ESLint will have fixed the usages of this in the test file, but it won't touch these comments.

Can you update all of these to match the projects code style. No quoted property names and single quote strings (i.e `firstName: 'John'`)